style/pages/projects-details.css

```diff
@@ -21,7 +24,78 @@
 
 .projects-details .related-projects-list .nav {
   overflow-y: auto;
-  height: calc(100vh - 130px);
+  height: calc(100vh - (var(--app-header-height) + 64px));
+}
+
+.projects-details .container {
+  --container-padding: 1.8rem;
+  gap: 1rem;
+  overflow-x: hidden;
+}
+
+.projects-details .project,
+.projects-details .project .checklist .tasks,
+.projects-details .activity {
+  display: flex;
+  flex-direction: column;
+}
+
+.projects-details .project {
+  gap: 2.5rem;
+}
+
+.projects-details .project .header,
+.projects-details .project .stats .content {
+  display: flex;
+  align-items: center;
+}
+
+.projects-details .project .header {
+  justify-content: space-between;
+}
+
+.projects-details .project .header .mini-profile .title {
+  font-size: 1.25rem;
+}
+
+.projects-details .project .stats {
+  overflow-x: auto;
+}
+
+.projects-details .project .stats .content {
+  gap: 5rem;
+}
+
+.projects-details .project .checklist .progress-bar {
+  margin-bottom: 1rem;
+}
+
+.projects-details .project .checklist .tasks {
+  gap: 0.5rem;
+}
+
+.projects-details .activity .tabs {
+  margin-left: calc(var(--container-padding) * -1);
+  margin-right: calc(var(--container-padding) * -1);
+  padding: 0 var(--container-padding);
+}
+
+.projects-details .activity .tabs .content-title {
+  margin-bottom: 0;
+}
+
+.projects-details .activity {
+  gap: 2rem;
+}
+
+.projects-details .activity .tabs .tab {
+  border: none;
+  background: transparent;
+}
+
+.projects-details .activity .tabs .tab.active,
```

Comment:
Don’t you have this already?

Reply:
In this context, it must be primary color,  and grey in index page

Reply:
No, actually not. It's designer's fault. Same elements should behave the same as much as possible, if it doesn't break their functionality. So here just choose whatever you want: either primary or grey